### ISSUE TITLE
non-parent-init-called false positive

### ISSUE THREAD

**Issue by pylint-bot** [PyCQA/pylint#308]
Originally reported by: **Claudiu Popa (BitBucket: [PCManticore](http://bitbucket.org/PCManticore), GitHub: @PCManticore)**

---

Given the following snippet, extracted from standard library's tracemalloc, a non-parent-init-called is emitted.

```
#!python

class _Traces(Sequence):
    def __init__(self, traces):
        Sequence.__init__(self)
        # traces is a tuple of trace tuples: see Trace constructor
        self._traces = traces
```

---
- Bitbucket: https://bitbucket.org/logilab/pylint/issue/308
